priv/data.yml

```diff
@@ -1,2 +1,3 @@
 - "No filme Carros da Pixar, o Relâmpago Marquinhos sofre um incidente durante a corrida final e vai parar no canteiro e o Doc Hudson, o treinardor, fala pra ele: Ache um ritmo cômodo e recupere essa volta. É muito bonito e profundo isso, mas bonito mesmo é PR revisado, no ritmo de vcs ♥"
 - "Aê pessoal lindo, tudo certo? Vocês sabiam que empresas como o Facebook e o Google têm setores de políticas públicas que promovem ações para ajudar e melhorar problemas sociais? Essas empresas têm promovido um conceito apelidado de JoMO que é Joy of Missing Out ou Alegria de perder, para quem não sabe inglês. O JoMO é você se sentir feliz de com o que está fazendo e aproveitar o seu tempo sem necessidade de se atualizar nas suas redes sociais, ou seja, o oposto do FoMO, Fear of Missing Out ou Medo de perder, que é um problema social moderno que causa um uso não saudável de redes sociais a todo momento. Mas já que todo mundo aqui tem FoMO forte, vítimas de nossa sociedade, o que vcs acham de não perder o review do meu PR?"
+- "Aê meus princesos e príncipas, tudo bem? Vocês sabiam que em 18 de dezembro de 1954 foi fundado no interior do Rio Grande do Sul o município de Não-Me-Toque? Este que tem apenas 17 mil habitantes e um PIB de aprox. R$ 462mil, aqui na empresa nós já fechamos contratos maiores, faturamos mais que uma cidade toda! Não-Me-Toque também me lembra do clássico noventista You Can't Touch This do MC Hammer que claramente foi inspirado pela pequena cidade rio grandense. Sabe o que é 18 também mas não é para dezembro? O review do meu PR, 18 linhas só po :gato_de_botas:"
```

Comment:
I think that this quote is very specific, for example "Sabe o que é 18 também mas não é para dezembro? O review do meu PR, 18 linhas só po"
Your PR needs to have only 18 lines to makes sense, you know? So IMHO you can close this PR 😆

Reply:
Or we can think in a way to change this last phrase 🤔 
Any ideas @CaiqueMitsuoka ?

Reply:
WDYT?
```
Mas muuuuito além de Não-Me-Toque e bem mais pertinho de você sabe o que dá para tocar? Isso meu o botão de `Start a Review` do meu PR.
```